Features/AdminCommands/UnBanCommand.cs

```diff
@@ -0,0 +1,37 @@
+using DSharpPlus.Commands;
+using DSharpPlus.Entities;
+using DSharpPlus.Commands.ContextChecks;
+using ArtcordAdminBot.Features.Helpers;
+
+namespace ArtcordAdminBot.Features.AdminCommands
+{
+    public partial class AdminCommandGroup
+    {
+        
+
+        [Command("Unban")]
+        [RequirePermissions(DiscordPermissions.BanMembers)]
+        public async Task BanAsync(CommandContext ctx, 
```

Comment:
Rename it to UnbanAsync, my guess is you forgot to rename it lol
```suggestion
        public async Task UnbanAsync(CommandContext ctx, 
```